pdns/Makefile.am

```diff
@@ -315,6 +316,7 @@ pdnsutil_SOURCES = \
 	pdnsutil.cc \
 	qtype.cc \
 	rcpgenerator.cc rcpgenerator.hh \
+	svc-records.cc svc-records.hh \
```

Comment:
you're the autoconf expert, but it looks weird sometimes having and sometimes _not_ having the `.hh` file in the `..._SOURCES` lists.

Reply:
You only need it in one place so it is disted, but it might make sense to specify it everywhere indeed